centos.org/pipelines/lib/foremanDuffyJob.groovy

```diff
@@ -0,0 +1,140 @@
+pipeline {
+    agent { label 'cico-workspace' }
+
+    options {
+        timestamps()
+        ansiColor('xterm')
+        buildDiscarder(logRotator(numToKeepStr: '3'))
+    }
+
+    environment {
+        ANSIBLE_CALLBACK_WHITELIST = 'community.general.opentelemetry'
+    }
+
+    stages {
+        stage('Setup Environment') {
+            steps {
+                deleteDir()
+                git url: 'https://github.com/theforeman/forklift.git'
+
+                sh(label: 'pip install', script: 'pip3.8 install --user opentelemetry-api opentelemetry-sdk opentelemetry-exporter-otlp python-cicoclient')
```

Comment:
Do we still need python-cicoclient? If we use duffy, it's redundant, right?
```suggestion
                sh(label: 'pip install', script: 'pip3.8 install --user opentelemetry-api opentelemetry-sdk opentelemetry-exporter-otlp')
```

Reply:
copy/waste ;)